sunpy/visualization/draw.py

```diff
@@ -0,0 +1,106 @@
+"""
+This module provides functions that draw on Map plots.
+"""
+import numpy as np
+from matplotlib.lines import Line2D
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+from sunpy.coordinates import HeliographicStonyhurst
+from sunpy.visualization import wcsaxes_compat
+
+__all__ = ["draw_equator", "draw_prime_meridian"]
+
+
+def draw_equator(axes, resolution=300, **kwargs):
+    """
+    Draws the equator as seen by the axes observer.
+
+    Parameters
+    ----------
+    axes : `matplotlib.axes.Axes`
+        The axes to plot the equator on.
+    resolution : `int`
+        The number of points used to represent the equator.
+
+    Examples
+    --------
+    >>> import sunpy.map
+    >>> from sunpy.visualization import draw
+    >>> import sunpy.data.sample   # doctest: +REMOTE_DATA
+    >>> aia = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)   # doctest: +REMOTE_DATA
+    >>> aia.plot()   # doctest: +SKIP
+    >>> draw.draw_equator(aia._check_axes(None))   # doctest: +SKIP
+    """
+    if not wcsaxes_compat.is_wcsaxes(axes):
+        raise ValueError('axes must be a WCSAxes')
+    axes_frame = axes._transform_pixel2world.frame_out
+    observer = axes_frame.observer
+
+    c_kw = {'linewidth': 1,
+            'color': 'white',
+            'zorder': 100}
+    c_kw.update(kwargs)
+
+    transform = axes.get_transform(axes_frame)
+    c_kw.setdefault('transform', transform)
+
+    lat = 0*u.deg
+    lat0 = SkyCoord(np.linspace(-90, 90, resolution) * u.deg,
```

Comment:
```suggestion
    lat0 = SkyCoord(np.linspace(-180, 180, resolution) * u.deg,
```
You need to construct the whole equator because you don't know where the observer is.